www/autonCreator.js

```diff
@@ -22,7 +22,7 @@ let waypointSelected = false;
 const WaypointAction = {
     MOVE: 1,
     ROTATE: 2,
-    SPLINE_ROTATE: 3,
+    SPLINE_ROTATE: 3, // Still needs implementation

```

Comment:
I believe the logic for this is at autonCreator.js:367, but wasn't created under this enumeration. We could redefine it to fall under here, or remove the enumeration from the list. I'd lean towards re-defining the section at 367, but if it's too coupled to other features we can leave it

Reply:
This  enum value should be gone from the latest version of this branch